docs-src/prod-readiness-context/data-encryption.md

```diff
@@ -0,0 +1,42 @@
+---
+id: data-encryption
+title: How to encrypt data using a custom Data Converter
+description: Using encryption in your custom Data Converter ensures that all your sensitive application data is secure when handled by the Temporal Server.
+sidebar_label: Data Encryption
+tags:
+  - guide-context
+  - production-readiness
+---
+
+Temporal stores and persists the data handled in your Workflow Execution.
+For example, any data that is persisted in the following contexts is stored in the Workflow Execution Event History:
+
+- inputs to your Workflows, Activities, and Child Workflows
+- inputs to your Signals
+- metadata information
+- results of Local Activities, Side Effects
+- Search Attributes
+- Application Errors and Failures.
+
+Encrypting this data at your application-level ensures that all your sensitive application data is secure when handled by the Temporal Server. It also ensures that your data exists unencrypted only on the Client and the Worker process that is executing the Workflows and Activities, on hosts that you control.
```

Comment:
Can't encrypt a Search Attribute or metadata. We probably should structure it differently.

Reply:
updated